### ISSUE TITLE
Building for OSX

### ISSUE THREAD

**Issue by xsyn** [wazuh/ossec-wazuh#24]
When trying to build for OSX (El Capitan in this case, but it will most versions), I get the following error:

```
 CC remoted/secure.o
remoted/secure.c:10:10: fatal error: 'sys/epoll.h' file not found
#include <sys/epoll.h>
```

It seems that epoll is Linux only, and for use on OSX kqueue is required. This issue seems to be an issue with this fork, as the standard OSSEC does compile and install.

I'd really like to be able to use your ELK stack integration within our enterprise, however we are mostly OSX machines. Is this something on your roadmap?

**Comment by snaow** [comment]
Hello @xsyn,

We have been working in some improvements to network protocols performance in OSSEC, adding a new one.
Using _epoll_ we have improved drastically network requests performance, bad news are, like you said, _epoll.h_ library is not installed by default in OSX, we will release a hotfix early next week to fix this issue, we prefer don't force users to install _kqueue library_.
I'll update this issue when the fix be done.


In the meantime you can install version [1.0.4 (Release)](https://github.com/wazuh/ossec-wazuh/releases/tag/v1.0.4), the version is fully compatible with v1.1, once we launch the hotfix you will be able to upgrade it easily.

Thanks for the feedback, we really appreciate it.

Best regards,

Pedro S.

**Comment by xsyn** [comment]
Thank you so much Pedro,

I'll give this a try.

**Comment by vikman90** [comment]
Hi @xsyn.

We did a change on the code that fixes the issue making use of kqueue. It seems that it's not necessary to install any additional package.

It's available on the development branch, and we'll include it in out next release.

Thank you very much for your feedback.
Kind regards.

Victor Fernandez.

**Comment by jessesanford** [comment]
Has this been merged yet? Any idea when mac support will be available again?

**Comment by vikman90** [comment]
Hi @jessesanford,

yes, that issue was merged into the version 1.1.1 (https://github.com/wazuh/wazuh/releases/tag/v1.1.1). You should be able to compile and install Wazuh on MacOS.

Best regards.
